src/collections/_documentation/workflow/issue-owners.md

```diff
@@ -3,57 +3,77 @@ title: 'Issue Owners'
 
-`type` can be either `path`, `url` or `tags.TAG_NAME`, depending on whether you’re specifying a path, URL or on a specific tag.
+When creating a rule from the issue page, you'll see some suggested paths and URLs based on the events in the issue. Note that Sentry doesn't suggest tags.
 
-`glob` will be the path or URL you are specifying (for example, `src/javascript/*` or `https://www.example.io/checkout`. You can use the _*_ character to match everything, or the _?_ character to match any single character. _This feature does not support regex._
+### Suggested Assignees
 
-`owner` can be the email of a Sentry user, or the name of a team, prefaced with `#` (i.e., `#backend-team`). To list multiple owners of the same path or URL, place them on the same line like so:
+On the issue page, you'll see suggested assignees based on ownership rules matching the event you're looking at (by default, the issue page shows the latest event). Suggested assignees can also be based on [suspect commits]({%- link _documentation/workflow/releases/index.md -%}#after-associating-commits). You can assign the *issue* to a suggested assignee by clicking on the suggestion. An event can have multiple suggested assignees if it matches multiple ownership rules.
 
-`type:glob owner1 owner2 owner3`
+[{% asset issue-owners/suggested_assignees.png alt="Avatar icons of suggested assignees and ability to click on them for assignment." width="400"%}]({% asset issue-owners/suggested_assignees.png @path %})
 
-Teams and users _must_ have access to the project to become owners. To grant a team access to a project, navigate to project settings > Project Teams, and click ‘Add Team to [project]’. To grant a user access to a project, the user must be a member of a team with access to the project. To add a user to a project’s team, navigate to Project Settings > Project Teams, select a team, then click ‘Add Member.’
+**Auto-Assign**
+
+You can automatically assign issues to their owners by enabling the following setting in **Project Settings > Issue Owners**.
+
+[{% asset issue-owners/new_event_matches.png alt="Toggle to automatically assign issues to their owners."%}]({% asset issue-owners/new_event_matches.png @path %})
+
+If an issue is already assigned, new events for that issue will not re-assign the issue even if they have a different owner. If an issue is not assigned, but a new event has multiple owners, Sentry assigns it to the owner matching the longest `pattern` in the rules that matched (regardless of the rule `type`).
+
+### Issue Alerts
+
+You can send [Issue Alerts]({%- link _documentation/workflow/alerts-notifications/alerts.md -%}#issue-alerts) to issue owners. Issue alerts are event-driven: when Sentry receives an event, it evaluates issue alert rules for the issue for that event. If the alert conditions match, Sentry sends an alert to the *event owners that triggered the alert*.
```

Comment:
"event owners that triggered the alert" -> "owners of the event that triggered the alert"